src/main/app/fees/feesClient.ts

```diff
@@ -1,82 +1,90 @@
 import * as config from 'config'
 import { request } from 'client/request'
 import { plainToClass } from 'class-transformer'
-
-import { CalculationOutcome } from 'app/fees/models/calculationOutcome'
-import { RangeGroup } from 'app/fees/models/rangeGroup'
-
 import { ClaimValidator } from 'app/utils/claimValidator'
+import { FeeOutcome } from 'fees/models/feeOutcome'
+import { FeeRange } from 'fees/models/feeRange'
 import { StringUtils } from 'utils/stringUtils'
-import { MoneyConverter } from 'app/fees/moneyConverter'
 
 const feesUrl = config.get('fees.url')
-
-const issueFeeCode = config.get<string>('fees.issueFee.code')
-const hearingFeeCode = config.get<string>('fees.hearingFee.code')
+const service: string = config.get<string>('fees.service')
```

Comment:
I don't think its worth specifying the type on both the left hand side and the RHS.
REC removing it from the left on all these fields